ibis/pandas/execution.py

```diff
@@ -752,6 +752,28 @@ def execute_between(op, data, lower, upper, scope=None):
     return data.between(lower, upper)
 
 
+@execute_node.register(
+    ops.Time,
+    pd.Series,
+)
+def execute_time(op, data, scope=None):
+    return op.to_expr()
+
+
+@execute_node.register(
+    ops.BetweenTime,
+    ir.TimeColumn,
+    (pd.Series, str, datetime.time),
+    (pd.Series, str, datetime.time),
+)
+def execute_between_time(op, data, lower, upper, scope=None):
+
+    # TODO(jreback), this is most likely wrong
```

Comment:
this node is hit correctly, BUT I cannot get the data at this point (IOW the timestampseries)
column gives me the name of the column, but I don't have access to the table to select.

currently the impl is wrong (its going to be something like this (if data is a Series)

``indexer = pd.DatetimeIndex(data).indexer_between_time(lower, upper)``

then some other code (i have to convert to a boolean array)